### ISSUE TITLE
[VPAT] Material 3 Expressive Widgets for VPAT

### ISSUE THREAD

**Issue by QuncCccccc** [flutter/flutter#185124]
As Material Design evolves, Material 3 Expressive (M3E) introduces 5 new components which should be added for VPAT assessment once they are fully implemented.

These new components are being added to the Flutter Material package as part of the M3E integration, detailed in the [Material 3 Expressive for Flutter design document](https://docs.google.com/document/d/15XBE5xraSiMh_Oep_hclrX8rnT7rhM69L1kaMN94gdo/edit?usp=sharing).

The new components to be added to the VPAT assessment list are:

- [ ] Button groups
- [ ] FAB menu
- [ ] Loading Indicator
- [ ] Split buttons
- [ ] Toolbars

**Comment by saurabh-mirajkar** [comment]
@QuncCccccc  Thanks for the proposal. Labelling for `team-accessibility` to investigate further.

**Comment by QuncCccccc** [comment]
I updated the issue description to include existing components that have m3e update. 
CC: @chunhtai
